### PR TITLE
#162199743  Display more information about articles

### PR DESCRIPTION
**Pull request by gitaumoses4**
**What does this PR do?**

Adds more information to the article cards

**Description of Task to be completed?**

Allow users to view the number of likes in the article lists
Allow the users to view the article ratings

**How should this be manually tested?**

Clone the repository to your local environment.
```
git clone https://github.com/andela/ah-codeblooded-frontend.git
```

Change directories
```
cd ah-codeblooded-frontend/
```

Install the dependencies
```
npm install
```

Run the server
```
yarn start
```

It will open a browser with the current route at `/`
The information about the article will be displayed on each card.

**What are the relevant pivotal tracker stories?**

[#162199743](https://www.pivotaltracker.com/story/show/162199743)

**Screenshots (if appropriate)**

screen shot 2018-11-27 at 04 18 35

### CODE REVIEW

**Review by YAHYA-H — APPROVED** [review]
👍

**Review by SrMoffat — COMMENTED** [review]
@gitaumoses4 ,
I always learn new things when looking at your code. The UI/UX has greatly improved after these changes.

File: src/components/ArticleCard/index.js [hunk 1]
```diff
@@ -26,6 +27,7 @@ class ArticleCard extends React.Component {
       <div className="col info-segment">
         {this.authorInfo()}
         {this.articleInfo()}
+        { this.articleRating() }
```
Comment (paulrimiru) [inline src/components/ArticleCard/index.js:30]: Remove the padding here

File: src/components/ArticleCard/index.js [hunk 2]
```diff
@@ -75,6 +78,37 @@ class ArticleCard extends React.Component {
...
   };
 
+  articleRating = () => {
+    const { article } = this.props;
+    const { likes, dislikes } = article.reactions;
+    return (
+      <div className="article-rating meta valign-wrapper">
+        {
+          article.avg_rating.avg_rating > 0 && (
+            <>
```
Comment (paulrimiru) [inline src/components/ArticleCard/index.js]: I like it that you are using React fragments, the syntax sugar it offers is great but for readability I prefer we use the `<Fragment>` tag

File: src/containers/ArticleListing/index.js [hunk 3]
```diff
@@ -97,17 +98,26 @@ export class ArticleListing extends React.Component {
           <div
             key={article && article.slug}
             className="slider-content"
-            style={{ background: `url('${(article && article.image) || placeholder}') no-repeat center center` }}
+            style={
```
Comment (paulrimiru) [inline src/containers/ArticleListing/index.js]: Avoid inline styles

**Review by paulrimiru — CHANGES_REQUESTED** [review]

File: src/components/ArticleCard/index.js (hunk 2 above)
Comment (gitaumoses4) [reply src/components/ArticleCard/index.js]: Cool. I will implement that.

File: src/components/ArticleCard/index.js (hunk 1 above)
Comment (gitaumoses4) [reply src/components/ArticleCard/index.js:30]: Okay. Thanks for the review.

File: src/containers/ArticleListing/index.js (hunk 3 above)
Comment (gitaumoses4) [reply src/containers/ArticleListing/index.js]: Thanks for the review @PaulKariukiRimiru 
 As you can see from this. The background image is dynamic (it is retrieved from every single article), in order for it to be displayed for each article. 
I could have used an  tag, but this won't work for the slider component. Kindly give me any suggestions you may have for this implementation.

Comment (paulrimiru) [reply src/containers/ArticleListing/index.js]: You can try and toggle use an `img` tag instead then you can toggle the image from the `src` prop, you can also apply styling to it as you did with the `div`

Comment (gitaumoses4) [reply src/containers/ArticleListing/index.js]: I have implemented these changes @PaulKariukiRimiru

File: src/components/ArticleCard/index.js (hunk 1 above)
Comment (gitaumoses4) [reply src/components/ArticleCard/index.js:30]: Done

File: src/components/ArticleCard/index.js (hunk 2 above)
Comment (gitaumoses4) [reply src/components/ArticleCard/index.js]: Done

**Review by paulrimiru — APPROVED** [review]

**Pull request merged**